tests/profiles-enabled/test_shipments.py

```diff
@@ -733,8 +733,8 @@ def side_effects(**kwargs):
             cert = kwargs['certificateId']
             return map_describe[cert]
 
-        with mock.patch('apps.shipments.serializers.boto3.client') as serial_client, \
-                mock.patch('apps.shipments.models.boto3.client') as model_client:
+        with mock.patch('apps.shipments.serializers.tracking_data.boto3.client') as serial_client, \
```

Comment:
I think if you use the wildcard imports in `__init__.py`s above, you can revert these changes